SegmentEditorSplitVolume/SegmentEditorSplitVolumeLib/SegmentEditorEffect.py

```diff
@@ -151,6 +172,8 @@ def onApply(self):
     slicer.app.setOverrideCursor(qt.Qt.WaitCursor)
     for segmentIndex in range(segmentationNode.GetSegmentation().GetNumberOfSegments()):

```

Comment:
For consistency, it should not be _all segments_ but _all visible segments_.

You can get visible segment IDs like this:

```
inputSegmentIDs = vtk.vtkStringArray()
segmentationNode = self.scriptedEffect.parameterSetNode().GetSegmentationNode()
segmentationNode.GetDisplayNode().GetVisibleSegmentIDs(inputSegmentIDs)
```